src/main/groovy/com/pstehlik/groovy/gelf4j/net/GelfTransport.groovy

```diff
@@ -118,92 +56,15 @@ class GelfTransport {
    * @return
    */
   private byte[] gzipMessage(String message) {
+
     def targetStream = new ByteArrayOutputStream()
-    def zipStream = new java.util.zip.GZIPOutputStream(targetStream)
-    zipStream.write(message.getBytes(USED_CHARSET))
+    def zipStream = new GZIPOutputStream(targetStream)
+    zipStream.write(message.getBytes('UTF-8'))
     zipStream.close()
     def zipped = targetStream.toByteArray()
     targetStream.close()
-    return zipped
-  }
-
-  /**
-   * Generate unique Message ID
-   * @return
-   */
-  private String generateMessageId() {
-    return new StringBuffer(36)
-      .append(format(getIP()))
-      .append(sep)
-      .append(format(getJVM()))
-      .append(sep)
-      .append(format(getHiTime()))
-      .append(sep)
-      .append(format(getLoTime()))
-      .append(sep)
-      .append(format(getCount())).toString()
-  }
-
-  protected String format(int intval) {
-    String formatted = Integer.toHexString(intval)
-    StringBuffer buf = new StringBuffer("00000000")
-    buf.replace(8 - formatted.length(), 8, formatted)
-    return buf.toString()
-  }
-
-  protected static String format(short shortval) {
-    String formatted = Integer.toHexString(shortval)
-    StringBuffer buf = new StringBuffer("0000")
-    buf.replace(4 - formatted.length(), 4, formatted)
-    return buf.toString()
-  }
-
-  /**
-   * Unique across JVMs on this machine (unless they load this class
-   * in the same quater second - very unlikely)
-   */
-  protected int getJVM() {
-    return JVM
-  }
+    zipped
 
-  /**
-   * Unique in a millisecond for this JVM instance (unless there
-   * are > Short.MAX_VALUE instances created in a millisecond)
-   */
-  protected short getCount() {
-    synchronized (this.class) {
-      if (counter < 0) counter = 0
-      return counter++
-    }
   }
 
-  /**
-   * Unique in a local network
-   */
-  protected int getIP() {
-    return IP
-  }
-
-  /**
-   * Unique down to millisecond
-   */
-  protected short getHiTime() {
-    return (short) (System.currentTimeMillis() >>> 32)
-  }
-
-  protected int getLoTime() {
-    return (int) System.currentTimeMillis()
-  }
-
-  protected String format(int intval, int lastDigits) {
-    String formatted = Integer.toHexString(intval)
-    formatted = formatted.substring(0, formatted.size() >= lastDigits ? lastDigits : formatted.size())
-    return formatted.padLeft(lastDigits, '0')
-  }
-
-  protected static String format(short shortval, int lastDigits) {
-    String formatted = Integer.toHexString(shortval)
-    formatted = formatted.substring(0, formatted.size() >= lastDigits ? lastDigits : formatted.size())
-    return formatted.padLeft(lastDigits, '0')
-  }
```

Comment:
This wasn't necessary anymore as the unique ID for message chunks is now based on a random UUID